components/odh-notebook-controller/controllers/notebook_webhook.go

```diff
@@ -290,9 +324,9 @@ func (w *NotebookWebhook) Handle(ctx context.Context, req admission.Request) adm
 		mutatedNotebook.ObjectMeta.Annotations[updatePendingAnnotation] = needsRestart.Reason
 	} else {
 		delete(mutatedNotebook.ObjectMeta.Annotations, updatePendingAnnotation)
-	}
-
-	// Create the mutated notebook object
+        }
+	
+        // Create the mutated notebook object
```

Comment:
@jiridanek no idea why the diff tools thinks there is a diff here ... all I did was rebase with main and resolve some merge conflicts